COVID19/src/components/Input.js

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
-import { View, TextInput } from 'react-native';
+import {View, TextInput} from 'react-native';
 import styles from '../utils/styles/loginAuthStyles/loginAuthStyles';
 import Icon from 'react-native-vector-icons/AntDesign';
-
+import colors from '../utils/colors';
 const Input = ({placeholderTxt, iconName, onChangeInput}) => {
-    return(
-        <View style={styles.inputContainer}>
-            <Icon name={iconName} color="gray" size={25} style={{marginStart: 15, alignSelf:'center'}}/>
-            <TextInput                 
-                placeholder={placeholderTxt}
-                secureTextEntry={placeholderTxt === 'Password' && true}
-                placeholderTextColor="gray"
-                onChange={onChangeInput}
-            />
-        </View>
-    );
-}
+  return (
+    <View style={styles.inputContainer}>
+      <Icon
+        name={iconName}
+        color={colors.white}
+        size={25}
+        style={{marginStart: 15, alignSelf: 'center'}}
```

Comment:
The in-line styles are not allowed

Reply:
fixed: the style is now in its separate file